static/app/routes.tsx

```diff
@@ -527,18 +527,15 @@ function buildRoutes(): RouteObject[] {
       path: 'teams/',
       name: t('Teams'),
       component: make(() => import('sentry/views/settings/project/projectTeams')),
-      deprecatedRouteProps: true,
```

Comment:
No changes in `projectSettingsChildren` other than removing `deprecatedRouteProps` and reordering. Reordered because the list was half in order of the tabs on the settings bar and half in alphabetical order. Now it's fully in order of settings tabs, with "miscellaneous" at the bottom. Happy to revert this diff if it's annoying for review.